pkg/models/deployment_upgrade_assistant_status_response.go

```diff
@@ -31,32 +31,29 @@ import (
 	"github.com/go-openapi/validate"
 )
 
-// RemoteClusterInfo Information about the remote cluster for cross-cluster search.
```

Comment:
Git renames are sometimes brilliant.

Reply:
wow, that makes it really hard to review 🤷